src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -193,39 +198,39 @@ func CreateAdmin(
 	}
 }
 
-func ContainerBrokers() []string {
+func ContainerBrokers() ([]string, []string) {
 	c, err := common.NewDockerClient()
 	if err != nil {
 		log.Debug(err)
-		return []string{}
+		return nil, nil
 	}
 	nodes, err := common.GetExistingNodes(c)
 	if err != nil {
 		log.Debug(err)
-		return []string{}
+		return nil, nil
+	}
+	if len(nodes) == 0 {
+		return nil, nil
 	}
-	grp := errgroup.Group{}
-	mu := sync.Mutex{}
+
 	addrs := []string{}
+	stopped := []string{}
 	for _, node := range nodes {
-		s := node
-		grp.Go(func() error {
-			mu.Lock()
-			defer mu.Unlock()
-			addrs = append(
-				addrs,
-				fmt.Sprintf(
-					"127.0.0.1:%d",
-					s.HostKafkaPort,
-				),
+		addr := fmt.Sprintf(
+			"127.0.0.1:%d",
+			node.HostKafkaPort,
+		)
```

Comment:
nit: This is used in a bunch of places, it would make sense to have it as a method on `NodeState` or sth.